.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
+omit =
+    */site-packages/*
+    config.py
+    */tests/*
+    */migrations/*
```

Comment:
EOF(No new line at the end of the file) missing here

Reply:
@geofrocker so this is not a python file, does it mean it also has to follow like python standards such as EOF?